papers.md

```diff
@@ -18,6 +18,8 @@ theory](https://inspirehep.net/record/1785309)," arXiv:2003.06413 [hep-lat]. (Ma
 
 - M. Borisyak, N. Kazeev, "[Machine Learning on data with sPlot background subtraction](https://inspirehep.net/record/1737258)," arXiv:1905.11719 [cs.LG]. (May 28, 2019)
 
+- S. Caron, T. Heskes, S. Otten and B. Stienen, "[Constraining the Parameters of High-Dimensional Models with Active Learning](https://arxiv.org/abs/1905.08628)," Eur. Phys. J. C 79, 944 (2019), arXiv:1905.08628 [cs.LG]. (May 19, 2019)
```

Comment:
```suggestion
- S. Caron, T. Heskes, S. Otten and B. Stienen, "[Constraining the Parameters of High-Dimensional Models with Active Learning](https://inspirehep.net/literature/1735784)," Eur. Phys. J. C 79, 944 (2019), arXiv:1905.08628 [cs.LG]. (May 19, 2019)
```

Reply:
We prefer to have the links go to INSPIRE rather than directly the arXiv, as INSPIRE is a good hub to get to all forms of the publication.